deploy/action.yml

```diff
@@ -97,14 +111,18 @@ runs:
       with:
         credentials_json: '${{ inputs.cloud_credentials }}'
 
-    - name: Deploy to GCP Bucket
+    # Rsync to GCS
+    - name: Rsync GCS
       if: ${{ inputs.cloud_service == 'gcp' }}
-      uses: 'google-github-actions/upload-cloud-storage@v0.8.0'
-      id: 'upload-file'
+      uses: ConfigureID/gh-actions/rsync@main
       with:
```

Comment:
@martinmoscovich pay attention to this part here. We have specify `ConfigureID/gh-actions/rsync@....`  in `deploy action`